DOCS/man/options.rst

```diff
@@ -5555,15 +5555,15 @@ them.
         Disable any dithering done by mpv.
     auto
         Automatic selection.
-        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
-        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        On ``--vo=gpu``: detected depth or 8 bpc otherwise
+        On ``--vo=gpu-next``: detected depth or 8 bpc (for SDR target) or no
+        dithering otherwise.
```

Comment:
> or no dithering otherwise.
I would remove that. In fact gpu_next will always dither to target format, unless it is explicitly disabled with dither_depth=-1. While `detected depth` means "wire" or "format" depth.
```
    if (opts->dither_depth < 0)
        pars->params.dither_params = NULL;
```

Reply:
but what will it dither to if this is not set?
https://github.com/mpv-player/mpv/blob/374470d471a49491bceb4e1241402e3e56ccb209/video/out/vo_gpu_next.c#L850-L854

This is what made me assume that it does no dithering.

Reply:
It dithers to the target format if not disabled.